content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a [month of busy development](https://github.com/OpenRA/OpenRA/pulse/monthly). Time to get back to testing.
+
+The major changes this time around include:
+
+  - A whole new in-game map editor with all the benefits that come from it.
```

Comment:
English could be improved a bit here too.  I suggest "A whole new in-game map editor, with many benefits over the old editor.".